san/batch.py

```diff
@@ -23,7 +23,8 @@ def execute(self):
         idxs = sorted([int(k.split('_')[1]) for k in res.keys()])
         for idx in idxs:
             key = "query_{}".format(idx)
```

Comment:
this var is unused

Reply:
Yes, sorry for that, I forgot, that it isn't needed anymore, when I have the transforming function. I've just removed it.